src/Gamepad/Controllers/babylon.windowsMotionController.ts

```diff
@@ -247,15 +246,16 @@ module BABYLON {
             let childMesh : AbstractMesh = null;
             for (let i = 0; i < meshes.length; i++) {
                 let mesh = meshes[i];
-                if (mesh.id === WindowsMotionController.MODEL_ROOT_NODE_NAME) {
-                    // There may be a parent mesh to perform the RH to LH matrix transform.
+
+                // There may be a parent mesh to perform the RH to LH matrix transform.
+                if (mesh.parent && mesh.parent.name === WindowsMotionController.GLTF_ROOT_TRANSFORM_NAME)

```

Comment:
The parent is now returned as part of the meshes list

Reply:
Ok - so does that mean we are guaranteed to find a mesh with a null parent? In which case I can drop the .name check altogether

Reply:
OK I updated this section, no longer testing for 'root' and it is working great :)